linkerd/app/outbound/src/tcp/logical.rs

```diff
@@ -66,13 +66,12 @@ where
 
             connect
                 .push_make_thunk()
-                .instrument(|t: &Endpoint| match t.tls.as_ref() {
-                    Conditional::Some(tls) => {
-                        debug_span!("endpoint", server.addr = %t.addr, server.id = ?tls.server_id)
-                    }
-                    Conditional::None(_) => {
-                        debug_span!("endpoint", server.addr = %t.addr)
-                    }
+                .instrument(|t: &Endpoint| {
+                    debug_span!(
+                        "endpoint",
+                        server.addr = %t.addr,
+                        server.id = t.tls.value().map(|tls| tracing::field::debug(&tls.server_id)),
```

Comment:
```suggestion
                        server.id = t.tls.value().map(|tls| tracing::field::display(&tls.server_id)),
```

I think this was debug because it was an option